scripts/project.js

```diff
@@ -1,10 +1,25 @@
 var projects = [];
```

Comment:
Entire contents of this file need to be wrapped in an IIFE and pass into the IIFE a module.  Example (function(module) { }.